site/pages/theme-topic-en.hbs

```diff
@@ -0,0 +1,108 @@
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+				<div class="col-md-6">
+					<h3><a href="#">[Subtopic hyperlink text]</a></h3>
+					<p>Summary of the information or tasks that can be accomplished on the sub-topic page. Remove prose or promotional messaging. Use action verbs.</p>
+				</div>
+			</div>
+		</section>
+		<div class="pull-right col-xs-12 col-md-4">
+			{{>more-info}}
+		</div>
+	</div>
+</div>
+<div class="container">
+	<section class="whtwedo">
+		<h2>What we are doing</h2>
+		<div class="row wb-eqht">
+			<section class="col-lg-4 col-md-6">
```

Comment:
Replace `<section>` by `<div>`

Reply:
-whtwedo does apply css. It affect <li>
-replaced section with div both en/fr

Reply:
Added a gc-whatwedo class to project (duplicate of whtwedo).  theme-topic now refers to this new class where needed